virtualization-and-containers/material/exercise_vm_containers_text.md

```diff
@@ -0,0 +1,175 @@
+# Working with Virtualization and Containers
+
+**This exercise sheet is work in progress**
+
+In this exercise we will work the virtualization and container techniques that we have seen in the lecture. We will set up virtual machines manually first and then automatize the process. Afterwards we will build out own containers.
+
+## Deadline
+
+Please finish the mandatory parts of the work before **18 November 2021 at 9:00**.
+
+## Prerequisites
+
+- Installed and working VirtualBox, Vagrant and Docker.
```

Comment:
You could link to the "homework" here.

Reply:
I added the link an rephrased that part a little bit.